pkg/lifecycle/terraform/daemonless.go

```diff
@@ -55,19 +57,24 @@ func (t *DaemonlessTerraformer) WithStatusReceiver(
 		PlanConfirmer: t.PlanConfirmer.WithStatusReceiver(statusReceiver),
 		Terraform:     t.Terraform,
 		Viper:         t.Viper,
+		FS:            t.FS,
 
 		Status: statusReceiver,
 	}
 }
 
 func (t *DaemonlessTerraformer) Execute(ctx context.Context, release api.Release, step api.Terraform, confirmedChan chan bool) error {
-	t.dir = step.Path
```

Comment:
kind of tangential -- I want to keep runtime state on the stack, not in struct fields if possible